tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -12,6 +12,23 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+$_dim: 10px;
+
+.notification-icon-button {
+  position: relative;
+}
+
+.red-dot {
+  background-color: mat-color($mat-red, 700);
+  border-radius: $_dim / 2;
+  height: $_dim;
+  position: absolute;
+  right: $_dim;
```

Comment:
the fact that `right` == `$_dim` was a coincidence but not by design, right? If not, could you tell me why it is?

Reply:
You're right. I misplaced it.